server/chat/backend/agent/tools/victorops_tool.py

```
@@ -0,0 +1,110 @@
+    return VictorOpsClient(api_id=creds["api_id"], api_key=creds["api_key"])
+
+
+# ---------------------------------------------------------------------------
+# Tool functions
+# ---------------------------------------------------------------------------
+
+
+def get_victorops_incidents(
+    limit: int = 20,
+    user_id: Optional[str] = None,
+    **kwargs,
```

Ruff (0.15.12)

[warning] Missing type annotation for `**kwargs`

(ANN003)